ZeeguuAPI/ZeeguuAPITests/ZeeguuAPITests.swift

```diff
@@ -320,6 +320,58 @@ class ZeeguuAPITests: XCTestCase {
 	}
 	
+	func testGetTranslationsForWord() {
+		print("Testing getting translations for word:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("Gipfeltreffen", context: "Unmittelbar vor dem Gipfeltreffen der Europäischen Union mit der Türkei spricht Parlamentspräsident Martin Schulz (SPD) Klartext - eine Vereinbarung von Flüchtlingskontingenten mit der Türkei sei kaum aussichtsreich.", url: "http://www.spiegel.de/politik/ausland/eu-tuerkei-gipfel-streit-um-fluechtlingskontingent-a-1065093.html") { (dict) -> Void in
+			
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
+			self.testLock.signal()
+		}
+		testLock.wait()
+	}
+	
+	func testGetTranslationsForOtherWord() {
+		print("Testing getting translations for another word:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("buchstäblich", context: "\"Was wir im Fernsehen gesehen haben, kommt nun buchstäblich bis an unsere Haustür\", sagte die CDU-Vorsitzende.", url: "http://www.spiegel.de/politik/deutschland/fluechtlinge-angela-merkel-spricht-von-historischer-bewaehrungsprobe-fuer-europa-a-1067685.html") { (dict) -> Void in
+			
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
+			self.testLock.signal()
+		}
+		testLock.wait()
+	}
+	
+	func testGetTranslationsForUmlautWord() {
+		print("Testing getting translations for umlaut word:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("über", context: "Trotz der geringen Zahl der Befragten ist die Studie über Göttingen hinaus interessant, da die Viertel typisch sind für Bezirke mit geringer Wahlbeteiligung.", url: "http://www.spiegel.de/politik/deutschland/nichtwaehler-studie-darum-ist-die-wahlbeteiligung-so-gering-a-1094499.html") { (dict) -> Void in
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
+			self.testLock.signal()
+		}
+		testLock.wait()
+	}
+	
+	func testGetTranslationsForSentence() {
+		print("Testing getting translations for sentence:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("Lassen sich doch nur im Wahlkampf blicken!", context: "Lassen sich doch nur im Wahlkampf blicken!", url: "http://www.spiegel.de/politik/deutschland/nichtwaehler-studie-darum-ist-die-wahlbeteiligung-so-gering-a-1094499.html") { (dict) -> Void in
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
+			self.testLock.signal()
+		}
+		testLock.wait()
+	}
+	
+	func testGetTranslationsForWordPair() {
+		print("Testing getting translations for word pair:")
+		ZeeguuAPI.sharedAPI().getTranslationsForWord("taucht auf", context: "Kaum ein Politikerbild taucht in der Studie des Instituts für Demokratieforschung so häufig auf wie das des Wahlkämpfers, der mit einem Stand, Luftballons und Bratwürsten auftaucht und anschließend wieder in der Versenkung verschwindet.", url: "http://www.spiegel.de/politik/deutschland/nichtwaehler-studie-darum-ist-die-wahlbeteiligung-so-gering-a-1094499.html") { (dict) -> Void in
+			XCTAssertNotNil(dict)
+			print("translation: ", dict)
+			self.testLock.signal()
+		}
+		testLock.wait()
+	}
+	
```

Comment:
BTW, I like the variety of the tests for translation! I'll use them in my testing slides of the next year ;) If I don't forget :))